message-generator-tests.sh

```diff
@@ -6,18 +6,34 @@ else
     RUST_LOG="debug"
 fi
 
-search_dir="../../test/message-generator/test/"
 message_generator_dir="./utils/message-generator/"
-
 cd $message_generator_dir
 cargo llvm-cov clean
 
+cd ../../roles
+cargo build -p mining-device
+cargo llvm-cov -p pool_sv2
+cargo llvm-cov -p jd_server
+cargo llvm-cov -p jd_client
+cargo llvm-cov -p translator_sv2 
+cargo llvm-cov -p mining_proxy_sv2 
+cd ./roles-utils
+cargo llvm-cov 
+cd ../../
+
+search_dir="../../test/message-generator/test/"
+
+cd $message_generator_dir
+cargo build
+
 for entry in `ls $search_dir`; do
     if [ "$entry" = "interop-jdc-change-upstream.json" ]; then
         echo "Skipping $entry"
         continue
     fi
 
+    sleep 10
```

Comment:
maybe add a comment justifying this sleep?

Reply:
in order to give time to every process to close successfully

Reply:
my suggestion was to add a comment to the shell script